cmd/ocsp-updater/main.go

```diff
@@ -490,7 +492,7 @@ func (updater *OCSPUpdater) getSubmittedReceipts(serial string) ([]string, error
 // a certificate and returns a list of the configured logs that are not
 // present. This is the set of logs we need to resubmit this certificate to in
 // order to obtain a full compliment of SCTs
-func (updater *OCSPUpdater) missingLogs(logIDs []string) []cmd.LogDescription {
+func (updater *OCSPUpdater) missingLogs(logIDs []string) ([]cmd.LogDescription, error) {
```

Comment:
You could avoid having an error return here by processing the `cmd.LogDescription` into a local struct type that includes pre-hashed LogID. I believe that's what Publisher does, and it's cleaner since it avoids using a config object as an internal representation (see Config Plan on the wiki).